grid.html

```diff
@@ -251,3 +251,44 @@ <h5>Configuration of node with JSON</h5>
     "maxSessions": 5
   }
 }</code></pre>
+
+<h3>Specifying the Port</h3>
+
+<p>The default TCP/IP port used by the hub is 4444.If another application on your computer is already using this port
+or if you already have a Selenium-Server running, you’ll see the following message in the log output.  </p>
+
+<pre><code>
+10:56:35.490 WARN - Failed to start: SocketListener0@0.0.0.0:4444
+Exception in thread "main" java.net.BindException: Selenium is already running on port 4444. Or some other service is.
+</code></pre>
+
+<p>If this occurs you can either shutdown the other process that is using port 4444, or you can tell Selenium-Grid to use a different port for its hub. 
+Use the -port option for changing the port used by the hub.</p>
+
+<pre><code>  java -jar selenium-server-standalone-<version>.jar -role hub -port <port> </code></pre> 
+
+Ex:
+<pre><code> java -jar selenium-server-standalone-3.13.0.jar -role hub -port 4441 </code></pre>
+
+<p>However if you want to check what process is using port 4444 or to see all other ports used by other running programs on your machine you can use the command. </p>
```

Comment:
This is slightly offtopic, please remove this.

Reply:
done